test/wallet/index.ts

```diff
@@ -186,6 +189,15 @@ describe('Wallet', function () {
       '00141BA006D3363D2FB2785E8DF4E44D3A49908780CB4FB51F6D217C08C021429F'
     const address = 'rhvh5SrgBL5V8oeV9EpDuVszeJSSCEkbPc'
 
+    const { sign: REQUEST_FIXTURES } = requests
+    const { sign: RESPONSE_FIXTURES } = responses
+    const wallet = Wallet.fromSeed('ss1x3KLrSvfg7irFc1D929WXZ7z9H')
+    const wallet2 = Wallet.fromSeed('shsWGZcmZz6YsWWmcnpfr6fLTdtFV')
+    const wallet3 = new Wallet(
+      '02F89EAEC7667B30F33D0687BBA86C3FE2A08CCA40A9186C5BDE2DAA6FA97A37D8',
+      '00ACCD3309DB14D1A4FC9B1DAE608031F4408C85C73EE05E035B7DC8B25840107A',
+    )
```

Comment:
This should be in a `beforeEach`, so that error handling is a little better if something breaks in one of these `Wallet` methods